src/vma/dev/ring_simple.h

```diff
@@ -104,7 +104,7 @@ class ring_simple : public ring_slave
 	struct ibv_comp_channel* get_tx_comp_event_channel() { return m_p_tx_comp_event_channel; }
 	int			get_ring_descriptors(vma_mlx_hw_device_data &data);
 	void			modify_cq_moderation(uint32_t period, uint32_t count);
-
+	int			ack_and_arm_cq(cq_type_t cq_type);
```

Comment:
Please consider removing `request_notification_blocking()` after introducing `ack_and_arm_cq()`. request_notification_blocking() was added just for vma_modify_ring() at https://github.com/Mellanox/libvma/commit/6b19d6e21ae4736b961242146dd4c1e151be3ec7.

Reply:
good catch
done